### PR TITLE
feat(tcf/ui): changes to Vendor List title styling

### PR DESCRIPTION
**Pull request by Osgar**
## Description
Changes to Vendor List title styling

## Solves ticket/s
https://jira.scmspain.com/browse/PSP-3463

## Expected behavior
The correct visualization of the fonts and their sizes according to the design

## Review steps
**Before**
image
image

**After**
image
image

### CODE REVIEW

File: components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import TcfSecondLayerDecisionGroup from '../tcf-secondLayer-decision-group'
 export default function TcfSecondLayerVendorList({
   i18n,
+  BaseClass,
```
Comment (alextremp) [inline components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js:6]: `baseClass`

File: components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js [hunk 2]
```diff
@@ -89,6 +90,7 @@ export default function TcfSecondLayerVendorList({
 
 TcfSecondLayerVendorList.propTypes = {
   groupBaseClass: PropTypes.string,
+  BaseClass: PropTypes.string,
```
Comment (alextremp) [inline components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js:93]: `baseClass`

File: components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js (hunk 1 above)
Comment (Osgar) [reply components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js]: @alextremp Done 👍

File: components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js (hunk 2 above)
Comment (Osgar) [reply components/tcf/ui/src/SecondLayer/components/tcf-secondLayer-vendorList/index.js]: @alextremp Done +1

**Pull request merged**